src/olympia/versions/models.py

```diff
@@ -549,7 +549,7 @@ def reset_nomination_time(self, nomination=None):
 
     @property
     def is_listed(self):
-        return self.addon.is_listed
+        return self.channel == amo.RELEASE_CHANNEL_LISTED
 
```

Comment:
I think it can be removed - it was the first change I made in this commit, before realizing that the permission classes should no longer depend on it (we shouldn't have permission classes calling `obj.is_listed` anymore, much less when `obj` is a `Version`).

I'll try removing it, see what happens.